src/celmath/geomutil.h

```diff
@@ -8,37 +8,43 @@
 // as published by the Free Software Foundation; either version 2
 // of the License, or (at your option) any later version.
 
-#ifndef _CELMATH_GEOMUTIL_H_
-#define _CELMATH_GEOMUTIL_H_
+#pragma once
+
+#include <cmath>
 
 #include <Eigen/Core>
 #include <Eigen/Geometry>
-#include <cmath>
 
 namespace celmath
 {
 
 template<class T> Eigen::Quaternion<T>
 XRotation(T radians)
 {
-    T halfAngle = radians * T(0.5);
-    return Eigen::Quaternion<T>(std::cos(halfAngle), std::sin(halfAngle), 0, 0);
+    using std::cos;
+    using std::sin;
```

Comment:
you use sin & cos only once. the change doesn't have any sense for me

Reply:
If we ever throw our own custom numeric type into this, it would enable argument-dependent lookup on any sin/cos methods defined in that type's namespace.

Reply:
sounds reasonable